public/components/FeatureAnywhereContextMenu/Container/styles.scss

```diff
@@ -0,0 +1,7 @@
+.context-menu {
+  &__flyout {
+    &.euiFlyout--medium {
+      width: 740px;
```

Comment:
I don't think its good practice to be hardcoding width values for the flyouts. What sort of problems were you having with the default medium/large sizes?

Reply:
we decide to align with Alerting part on styling. So all the css styling was provided by Alerting team